app/data/models/dog.js

```diff
@@ -11,6 +11,10 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.UUID,
       allowNull: true
     },
+    index_number_base: {
+      type: DataTypes.INTEGER,
+      allowNull: true
```

Comment:
`index_number_base` isn't nullable in the DB. It is also unique so probably could do with adding that here.

Reply:
Sorry, typo